### PR TITLE
Add invariant semantic slot for signal

### PR DESCRIPTION
**Pull request by leddie24**
Change newSignal color to themePrimary

screen shot 2017-11-10 at 4 03 56 pm

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

- Adds invariant semantic slot for signal.
- Change color of newItem signal

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/styling/src/interfaces/ISemanticColors.ts [hunk 1]
```diff
@@ -131,6 +131,10 @@ export interface ISemanticColors {
    * Background for success
    */
   successBackground: string;
+  /**
+   * The color for signals
+   */
+  signal: string;
```
Comment (dzearing) [inline packages/styling/src/interfaces/ISemanticColors.ts]: I'm not sure this makes sense as a semantic slot. It is extremely specific. @phkuo Is there an alternative?

**Comment by phkuo** [comment]
@leddie24 are we sure that should be unthemed? It definitely looks like it should theme along with everything else... What's the justification for leaving it unthemed?

What's the concept here? What other controls share a similar concept to this?

**Comment by leddie24** [comment]
@phkuo, there's an email thread regarding this decision; I'll forward this to you and Zearing.

File: packages/office-ui-fabric-react/src/common/_semanticSlots.scss [hunk 2]
```diff
@@ -18,6 +18,7 @@ $blockingBackgroundColor: "[theme:blockingBackground, default: rgba(234,67,0,.2)
 $warningBackgroundColor: "[theme:warningBackground, default: rgba(255,185,0,.2)]";
 $warningHighlightColor: "[theme:warningHighlight, default: #ffb900]";
 $successBackgroundColor: "[theme:successBackground, default: rgba(186,216,10,.2)]";
+$signal: "[theme:signal, default: #0078d7]";
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/common/_semanticSlots.scss]: please following naming convention and have this be $signalColor

File: packages/styling/src/interfaces/ISemanticColors.ts [hunk 3]
```diff
@@ -131,6 +131,10 @@ export interface ISemanticColors {
    * Background for success
    */
   successBackground: string;
+  /**
+   * The color for signals
```
Comment (phkuo) [inline packages/styling/src/interfaces/ISemanticColors.ts]: Please expand on what a signal is. Used for iconography indicating common states for things, etcetc.

File: packages/styling/src/styles/theme.ts [hunk 4]
```diff
@@ -101,6 +101,7 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean): ISema
     warningBackground: !isInverted ? 'rgba(255, 185, 0, .2)' : 'rgba(255, 251, 0, .6)',
     warningHighlight: !isInverted ? '#ffb900' : '#fff100',
     successBackground: !isInverted ? 'rgba(186, 216, 10, .2)' : 'rgba(186, 216, 10, .4)',
+    signal: '#0078d7',
```
Comment (phkuo) [inline packages/styling/src/styles/theme.ts]: add a newline above this to separate it from the error/warning stuff

**Comment by leddie24** [comment]
Closing PR since it seems like we want to keep signals themed for the time being.  Will re-visit this if this is not the case in a few weeks.

**Pull request closed without merging**